package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "Gulp build certbot.",
   "scripts": {
-    "test": "htmlproofer ./_site --empty-alt-ignore true --only-4xx true --file-ignore /_site/docs/_modules/,./_site/docs/api/display.html,./_site/docs/contributing.html,./_site/docs/genindex.html,./_site/docs/intro.html,./_site/docs/py-modindex.html,./_site/docs/search.html --allow-hash-href true"
+    "test": "htmlproofer ./_site --empty-alt-ignore true --only-4xx true --file-ignore /_site/docs/_modules/,./_site/docs/api/display.html,./_site/docs/api/le_util.html,./_site/docs/contributing.html,./_site/docs/genindex.html,./_site/docs/intro.html,./_site/docs/py-modindex.html,./_site/docs/search.html --allow-hash-href true"
```

Comment:
In the certbot master branch, `api/le_util.html` recently moved to `api/util.html`.  But this should be correct if the website builds docs from a release tag rather than master?